examples/cloud-composer-examples/composer_http_post_example/make_iap_request.py

```diff
@@ -1,151 +0,0 @@
-# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
this is now in the instructions to be downloaded from the docs samples repo where it is maintained.